Add details for recurrenceEditMode; remove redundant text from Scheduler -> config -> editing (#1070)
DevExpress/devextreme-documentation: api-reference/10 UI Widgets/dxScheduler/1 Configuration/recurrenceEditMode.md

Specifies the edit mode for recurring appointments.

---
This option accepts the following values:

- *"series"*  

>  Enables a user to edit only the entire appointment series.

Reviewer: ```suggestion
 Enables a user to edit only entire appointment series.
```

- *"occurrence"*  

>  Enables a user to edit only the individual appointment instance.

Reviewer: ```suggestion
 Enables a user to edit only individual appointment instances.
```

- *"dialog"*
 Displays a dialog that suggests to a user to choose between editing the entire series or only the individual instance.

> The Scheduler handles changes made to an instance and a series differently.

Reviewer: ```suggestion
The **Scheduler** handles changes made to an instance and a series differently.
```

> Changes made to recurring appointment instance and series are handled differently.

Reviewer: ```suggestion
The **Scheduler** handles changes made to an instance and changes made to a series differently.
```

Author: The **Scheduler** handles changes made to an instance and a series differently. Is it better?

Reviewer: Ok

- A new data object is created.       
This object contains the edited instance's data. The [onAppointmentAdding](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/onAppointmentAdding.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onAppointmentAdding') and [onAppointmentAdded](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/onAppointmentAdded.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onAppointmentAdded') event handlers are executed.

> When an individual appointment instance is deleted, the **Scheduler** adds it to the entire appointment series exceptions by updating the field that **recurrenceExceptionExpr** specifies. Because this is an update, the **onAppointmentUpdating** and **onAppointmentUpdated** event handlers are executed instead of [onAppointmentDeleting](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onAppointmentDeleting) and [onAppointmentDeleted](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onAppointmentDeleted).

Reviewer: ```suggestion
When a user deletes an instance, the **Scheduler** adds it to series' exceptions by updating the field specified by **recurrenceExceptionExpr**. Because this is an update, the **onAppointmentUpdating** and **onAppointmentUpdated** event handlers are executed instead of [onAppointmentDeleting](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onAppointmentDeleting) and [onAppointmentDeleted](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#onAppointmentDeleted).
```

Author: "...by updating the field specified by recurrenceExceptionExpr..." -> "...by updating the field specified **in** recurrenceExceptionExpr..." ?

Reviewer: I don't see much difference but okay, we can use "in" to remove the repetition of "by".
